frontend/src/components/drawer.jsx

```diff
@@ -88,6 +92,28 @@ const useStyles = makeStyles(theme => ({
 }));
 
 export default function MiniDrawer() {
+
+  //#region mockImageForTest
+  const mockImage = {
+    user: "www.instagram.com/Bob",
+    post: "www.instagram.com/Bob",
+    url: "https://i.redd.it/z9l08cn8wde41.png",
+    repost: "false",
+    description: "This is the caption",
+    comments: [
+      {
+        user: "Mia",
+        comment: "Nice pic"
+      },
+      {
+        user: "Khalifa",
+        comment: "Cool pic"
+      },
+    ],
+    date: "jan 23 2020"
+  }
+  //#endregion
```

Comment:
Please look at the image_template documentation. This is not a representation of the actual object and will not work.

Reply:
It should be fine considering that the variables are assign in the frontend

Reply:
Ok great to know